packages/apps/desktop-viewer-test/package.json

```diff
@@ -32,51 +32,54 @@
-    "@itwin/appui-abstract": "^4.7.3",
-    "@itwin/appui-layout-react": "^4.3.0",
-    "@itwin/appui-react": "^4.3.0",
-    "@itwin/unified-selection": "~1.1.2",
-    "@itwin/unified-selection-react": "^1.0.0",
-    "@itwin/components-react": "^4.3.0",
-    "@itwin/core-backend": "^4.7.3",
-    "@itwin/core-bentley": "^4.7.3",
-    "@itwin/core-common": "^4.7.3",
-    "@itwin/core-electron": "^4.7.3",
-    "@itwin/core-frontend": "^4.7.3",
-    "@itwin/core-geometry": "^4.7.3",
-    "@itwin/core-i18n": "^4.7.3",
-    "@itwin/core-markup": "^4.7.3",
-    "@itwin/core-orbitgt": "^4.7.3",
-    "@itwin/core-quantity": "^4.7.3",
-    "@itwin/core-react": "^4.3.0",
-    "@itwin/core-telemetry": "^4.7.3",
+    "@itwin/appui-abstract": "^5.0.0",
+    "@itwin/appui-react": "^5.5.0",
+    "@itwin/unified-selection": "^1.3.0",
+    "@itwin/components-react": "^5.5.0",
+    "@itwin/core-backend": "^5.0.0",
+    "@itwin/core-bentley": "^5.0.0",
+    "@itwin/core-common": "^5.0.0",
+    "@itwin/core-electron": "^5.0.0",
+    "@itwin/core-frontend": "^5.0.0",
+    "@itwin/core-geometry": "^5.0.0",
+    "@itwin/core-i18n": "^5.0.0",
+    "@itwin/core-markup": "^5.0.0",
+    "@itwin/core-orbitgt": "^5.0.0",
+    "@itwin/core-quantity": "^5.0.0",
+    "@itwin/core-react": "^5.5.0",
     "@itwin/desktop-viewer-react": "workspace:*",
-    "@itwin/ecschema-metadata": "^4.7.3",
-    "@itwin/ecschema-rpcinterface-common": "^4.7.3",
-    "@itwin/ecschema-rpcinterface-impl": "^4.7.3",
-    "@itwin/electron-authorization": "^0.15.0",
-    "@itwin/express-server": "^4.7.3",
+    "@itwin/ecschema-metadata": "^5.0.0",
+    "@itwin/ecschema-rpcinterface-common": "^5.0.0",
+    "@itwin/ecschema-rpcinterface-impl": "^5.0.0",
+    "@itwin/electron-authorization": "^0.21.0",
+    "@itwin/express-server": "^5.0.0",
     "@itwin/imodel-browser-react": "^1.0.0",
-    "@itwin/imodel-components-react": "^4.3.0",
-    "@itwin/imodels-access-backend": "^5.2.1",
-    "@itwin/imodels-access-frontend": "^5.2.1",
+    "@itwin/imodel-components-react": "^5.5.0",
+    "@itwin/imodels-access-backend": "^6.0.1",
+    "@itwin/imodels-access-frontend": "^6.0.1",
+    "@itwin/imodels-client-authoring": "^6.0.1",
     "@itwin/itwinui-css": "^1.12.0",
     "@itwin/itwinui-icons-react": "^2.4.0",
     "@itwin/itwinui-illustrations-react": "^2.0.1",
     "@itwin/itwinui-layouts-css": "^0.2.0",
     "@itwin/itwinui-layouts-react": "^0.2.0",
-    "@itwin/itwinui-react": "^2.12.0",
+    "@itwin/itwinui-react": "^3.18.0",
     "@itwin/itwinui-variables": "^2.0.0",
-    "@itwin/measure-tools-react": "^0.23.2",
-    "@itwin/presentation-backend": "^4.7.3",
-    "@itwin/presentation-common": "^4.7.3",
-    "@itwin/presentation-components": "^5.0.0",
-    "@itwin/presentation-frontend": "^4.7.3",
+    "@itwin/measure-tools-react": "latest",
+    "@itwin/object-storage-azure": "^3.0.0",
+    "@itwin/object-storage-core": "^3.0.0",
+    "@itwin/object-storage-google": "^3.0.0",
```

Comment:
I believe these 3 are not required with the latest release of `@itwin/imodels-access-backend`

Reply:
Also applies to `@itwin/cra-template-desktop-viewer` template